pkg/apis/bigquery/v1/types.go

```diff
@@ -0,0 +1,31 @@
+package v1
+
+import "time"
+
+type Job struct {
+	ID                          int64
+	JobName                     string
+	GCSBucketName               string
+	GCSJobHistoryLocationPrefix string
+	CollectDisruption           bool
+	CollectTestRuns             bool
+	Platform                    string
+	Network                     string
+	IPMode                      string
+	Topology                    string
+	Release                     string
+	FromRelease                 string
+	RunsUpgrade                 bool
+	RunsE2EParallel             bool
+	RunsE2ESerial               bool
+}
+
+type JobRun struct {
+	Name       string
+	JobName    string
+	Status     string
+	StartTime  time.Time
+	EndTime    time.Time
+	ReleaseTag string
+	Cluster    string
+}
```

Comment:
Are these an actual API type we'd return, or something we'd work with only transiently to merge the data with the existing testgrid data?

If it's the latter, suggest taking these out of the apis package. If not then disregard. :)

Reply:
It's an intermediate type used to marshal the bigquery rows. We have similar for TestGrid, which is why I put it here: https://github.com/openshift/sippy/tree/master/pkg/apis/testgrid/v1